package.json

```diff
@@ -47,6 +47,7 @@
     "@ably/vcdiff-decoder": "1.0.4",
     "@testing-library/react": "^13.3.0",
     "@types/crypto-js": "^4.0.1",
+    "@types/jmespath": "^0.15.2",
```

Comment:
we should add `jmespath` itself as an explicit dependency too, at the moment it's transitively installed through the aws sdk